packages/website/src/css/custom.css

```diff
@@ -72,19 +83,19 @@ html[data-theme='dark'] .header-github-link:before {
 }
 
 .theme-doc-sidebar-item-link-level-1 > a,
-.theme-doc-sidebar-item-category-level-1 > a {
-  font-size: 1rem;
+.theme-doc-sidebar-item-category-level-1 > .menu__list-item-collapsible > a {
```

Comment:
Typically we don't recommend targeting infima classes (I think you are following the thread in https://github.com/facebook/docusaurus/pull/6053 as well). I didn't check the site DOM so not sure if this is necessary; perhaps we need to add more stable class names if you can't target the right element?

Reply:
sadly it is, otherwise this is not working anymore :(

Reply:
Ah, I see why🙈I think I will add another class name for the category label itself